stacks-common/src/util/hash.rs

```diff
@@ -109,23 +113,23 @@ impl_array_newtype!(Keccak256Hash, u8, 32);
 impl_array_hexstring_fmt!(Keccak256Hash);
 impl_byte_array_newtype!(Keccak256Hash, u8, 32);
 
-#[derive(Serialize, Deserialize)]
-pub struct Sha256Sum(
-    #[serde(
-        serialize_with = "Hash32::json_serialize",
-        deserialize_with = "Hash32::json_deserialize"
-    )]
-    pub [u8; 32],
-);
-impl_array_newtype!(Sha256Sum, u8, 32);
-impl_array_hexstring_fmt!(Sha256Sum);
-impl_byte_array_newtype!(Sha256Sum, u8, 32);
-
-impl Default for Sha256Sum {
-    fn default() -> Self {
-        Sha256Sum::zero()
-    }
-}
+// #[derive(Serialize, Deserialize)]
```

Comment:
Please drop commented-out code

Reply:
I've now deleted all commented code I could find and remember. Let me know if you find anything else.